docs/application/web/guides/app-management/app-communication.md

```diff
@@ -4,17 +4,17 @@ The application data exchange features cover the various methods you have to com
 
 You can use the following application data exchange features in your Web applications:
 
-- [Message Port](./message-port.md)
+- [Message Port](message-port.md)
 
   Applications communicate with each other using message port communication. You can send and receive messages using the map data format and trusted or untrusted message port instances.
 
-- [Data Control](./data.md)
+- [Data Control](data.md)
```

Comment:
Current file name for data.md is data-control.md.

Reply:
Done.